cmake/scripts/windows/tools/patch.cmake

```diff
@@ -70,8 +70,8 @@ else()
   endif()
 
   # make sure that cmake can find the copied patch.exe
-  find_program(PATCH_FOUND NAMES patch patch.exe)
-  if(NOT PATCH_FOUND)
+  find_program(OWN_PATCH_FOUND NAMES patch patch.exe)
+  if(NOT OWN_PATCH_FOUND)
```

Comment:
Add a correct `find_package_handle_standard_args` check instead of this hack.
Also now this defines either `PATCH_EXECUTABLE` or `OWN_PATCH_FOUND ` which contains the patch executable path, but the fallback path (`OWN_PATCH_FOUND`) isn't used anywhere.

Reply:
I thought to fix Leia not to rewrite them.

Reply:
Is updated